index.js

```diff
@@ -3,126 +3,223 @@ const express = require('express'),
   morgan = require('morgan'),
   bodyParser = require('body-parser'),
   uuid = require('uuid');
+  const mongoose = require('mongoose');
+  const Models = require('./models.js');
 
 const app = express();
+const Movies = Models.Movie;
+const Users = Models.User;
+mongoose.connect('mongodb://localhost:27017/myFlixDatabase', {
+  useNewUrlParser: true, // Connect the DB (to perform CRUD)
+  useUnifiedTopology: true
+});
 
 app.use(morgan('common'));
 app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true}));
```

Comment:
Alternatively express supports json. So instead of bodyParser we could just replace these two lines with ,
`app.use(express.json());`